modules/orchestrator/src/main/java/org/eclipse/xpanse/modules/orchestrator/deployment/DeployTask.java

```diff
@@ -43,8 +43,8 @@ public class DeployTask {
     private UUID serviceTemplateId;
 
     /**
-     * The action of the deployment task.
+     * The action of to the deployment task.
      */
-    private DestroyScenario destroyScenario;
+    private DeploymentScenario deploymentScenario;
```

Comment:
any specific reason for adding this? We already have three different methods -> deploy, destroy and modify. So we already know the deployment scenario.

Reply:
@Alice1319 why do we need this? currently i see that the value for this is set only in our unit test. It is not used anywhere else. 
If it is not needed, we can delete the enum completely